src/gui/torrentcontentmodelfile.h

```diff
@@ -33,19 +33,21 @@
 
 #include "torrentcontentmodelitem.h"
 
-class TorrentContentModelFile : public TorrentContentModelItem
+class TorrentContentModelFile: public TorrentContentModelItem
 {
 public:
-  TorrentContentModelFile(const QString &fileName, qulonglong fileSize,
-                          TorrentContentModelFolder* parent, int file_index);
+    TorrentContentModelFile(const QString &fileName, qulonglong fileSize,
+                            TorrentContentModelFolder* parent, int file_index);
 
-  int fileIndex() const;
-  void setPriority(int new_prio, bool update_parent = true);
-  void setProgress(qreal progress);
-  ItemType itemType() const { return FileType; }
+    int fileIndex() const;
+    void setPriority(int new_prio, bool update_parent = true);
```

Comment:
I hope that you read my previous response and I don't have to write it for each such your excuses.
If you do something - do it completely.